superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -955,6 +955,18 @@ export const controls = {
     description: 'Whether to include a time filter',
   },
 
+  filter_order: {
+    type: 'SelectControl',
+    multi: true,
```

Comment:
By default, filters will be sorted by metric in descending order, if not select any`filter_order ` related to them.